doc/news/_preparation_next_release.md

```diff
@@ -34,11 +34,45 @@ docker run -it elektra/elektra
 ## Highlights
 
 - `kdb` now prohibits write operations on cascading keys that miss a corresponding existing key. See the details in the `Tools` section below and the new subsection on cascading writes in the [tutorial](/doc/tutorials/cascading.md) on cascading keys for further information. _(Alexander Firbas)_
-- <<HIGHLIGHT1>>
 - <<HIGHLIGHT2>>
 - <<HIGHLIGHT3>>
 
-### <<HIGHLIGHT1>>
+### Redshift and Elektra
+
+We have created a version of [Redshift](https://github.com/jonls/redshift/) that uses Elektra for configuration management!
+
+We removed and refactored Redshift's code for loading configuration files, parsing CLI options and validating configuration to use Elektra. Redshift with Elektra has nearly 700 fewer lines of code (-17%) and is a great example of what Elektra is all about: More applications with less code for configuration management!
+
+To test it, clone the [elektrify branch](https://github.com/qwepoizt/redshift/tree/improve-config/elektrify) and follow the instructions provided in [CONTRIBUTING.md](https://github.com/qwepoizt/redshift/blob/improve-config/elektrify/CONTRIBUTING.md).
+
+#### Benefits of Redshift using Elektra
+
+Refactoring Redshift to use Elektra brings the following benefits:
+
+- Fewer lines of code (~700 less LOC, 17%).
```

Comment:
At least once you should mention that the 17% is about configuration related code, not the total code.

Reply:
Thanks for pointing that out. I've made the calculation more transparent and updated the numbers accordingly.